src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -338,7 +336,14 @@ internal void StartProcess(string workerId, Process process)
                 // TODO: per language stdout/err parser?
                 if (e.Data != null)
                 {
-                    _logger.LogInformation(e.Data);
+                    if (e.Data.ToLower().Contains("warn"))
```

Comment:
nit: remove `ToLower` and use the `.Contains("warn", StringComparer.OrdinalIgnoreCase)`